mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -202,6 +206,13 @@ def process_wolfram_string(text):
         text = re.sub(r"!", r",factorial", text)
         return text
 
+    @staticmethod
+    def __find_num(pods, pod_num):
+        for pod in pods:
+            if pod.node.attrib['position'] == pod_num:
```

Comment:
Does a `pod` object **always** has `node` and `attrib` so we can get `position` without breaking?